test/gtest/ucp/test_ucp_atomic.cc

```diff
@@ -102,6 +119,28 @@ void test_ucp_atomic::nb_post(entity *e,  size_t max_size, void *memheap_addr,
     *(T*)&expected_data[0] = atomic_op_val<T, OP>(val, prev);
 }
 
+template <typename T, ucp_atomic_op_t OP>
+void test_ucp_atomic::nbx_post(entity *e,  size_t max_size, void *memheap_addr,
+                               ucp_rkey_h rkey, std::string& expected_data)
+{
+    ucs_status_t status;
+    T val, prev;
+
+    prev   = *(T*)memheap_addr;
+    val    = (T)ucs::rand() * (T)ucs::rand();
+
+    status = test_ucp_atomic::ucp_atomic_post_nbx<T>(e->ep(), OP, val,
```

Comment:
do we need test for "fetch"?

Reply:
not really, because all current fetch goes thru new API (we updated `ucp_atomic_fetch_nb` implementation to use `ucp_atomic_op_nbx`)

Reply:
ok